RayLin.md

```diff
@@ -1,22 +1,22 @@
-# 07/04 ~ 07/08
+# 07/11 ~ 07/15
 
-- [Firefox for Android]
-  - Bug 1086911 - Update style of about:firefox
-    - wait :antlam reply, will NI again this week
-  - Bug 1079504 - Improve about:addons if user has no add-ons installed
-    - feedback-
-    - NI pwalmsley
-  - Bug 1027535 - Addon Manager doesn't provide enough details in case of blocklisted addons
-    - provided screenshot but blocked by add-ons signing meta bug: Bug 1170113
-  - Bug 1282678 - [webvtt] Consider adding closed caption button to mobile video control
-    - redirected to Barbara to prioritize
-- [Firefox]
+- [Video Control]
   - Bug 1281414 - [webvtt] Update the design of closed caption button icon
-    - NI :shorlander
-  - Bug 1198279 - Closing Find Toolbar should clear all highlighted matches
-    - FIXED
+    - CC'd Carol, attcah comparison screenshot and reiterate intention
+  - Bug 883122 - [webvtt] Implement the rules for constructing a chapter tree from a text track
+    - build test page and study algorithm
```

Comment:
no, just had some discussion with Benjamin about algorithm, and no further plan for this bug at present.